inductiva/resources/machine_groups.py

```diff
@@ -54,7 +54,8 @@ class BaseMachineGroup(ABC):
     provider: Union[ProviderType, str] = "GCP"
     threads_per_core: int = 2
     data_disk_gb: int = 10
-    max_idle_time: Optional[Union[datetime.timedelta, int]] = None
+    max_idle_time: Union[datetime.timedelta,
+                         int] = datetime.timedelta(minutes=3)
```

Comment:
You can use `max_idle_time` as an int. It should make this code smaller.
```suggestion
                         int] = 3
```